cypress/integration/playedCards_spec.js

```diff
@@ -25,7 +18,8 @@
 //     it('gives list of played cards', function() {
 //         cy.visit('http://localhost:8080/dashboard')
     
-//         console.log(cy.get('#played-cards'))
+//         cy.get('#played-cards')
+//         cy.get('.error').should('eq', ') 
 //     })
 // })
 // des
```

Comment:
You can get rid of this file if you're just commenting it out. I'm gonna push it from my pr.

Reply:
cool will do thanks